Apps/Database/src/Delegates/DbDelegationDelegate.cs

```diff
@@ -82,6 +83,15 @@ public async Task<(IList<string> Hdids, int TotalHdids)> GetProtectedDependentHd
             return (records, totalCount);
         }
 
+        /// <inheritdoc/>
+        public async Task<Delegation?> GetDelegationAsync(Guid id)
+        {
+            IQueryable<Delegation> query = this.dbContext.Delegation
+                .Where(d => d.Id == id);
+
+            return await query.SingleOrDefaultAsync();
```

Comment:
CancellationToken should be passed all the way down.

Reply:
Changed.